cores/esp8266/umm_malloc/umm_malloc.c

```diff
@@ -550,44 +551,45 @@
 #undef DBG_LOG_FORCE
 
 /* ------------------------------------------------------------------------- */
+#define printf_P(fmt, ...)  do { static const char fstr[] PROGMEM = fmt; char rstr[sizeof(fmt)]; for (size_t i=0; i<sizeof(rstr); i++) rstr[i] = fstr[i]; printf(rstr, ##__VA_ARGS__); } while (0)
```

Comment:
Since we're doing this, can we also `#define printf(...) printf_P(__VA_ARGS__)`? It will make it much easier to upgrade umm to upstream version, if we ever want to do that.